microsoft-365/commerce/buy-or-edit-an-add-on.md

```diff
@@ -23,7 +23,7 @@ description: "Learn how to buy and manage add-ons for your Microsoft 365 for bus
 ms.date: 04/02/2021
 ---
 
-# Buy or manage add-ons
+# Buy or manage Microsoft 365 for business plans add-ons
```

Comment:
@lhcole613 - What do you think about switching the words around so it sounds less awkward? "Buy or manage add-ons for Microsoft 365 business plans"

Reply:
As long as it's OK not to use Microsoft 365 for business in this case (the official name), that's much better and I've made the change.

Reply:
@lhcole613 Please leave this title as-is